src/hooks/useEditVideoSheet.tsx

```diff
@@ -42,7 +45,9 @@ export const EditVideoSheetProvider: React.FC = ({ children }) => {
   const [sheetState, setSheetState] = useState<EditVideoSheetState>('closed')
   const [cachedSheetState, setCachedSheetState] = useState<EditVideoSheetState>('closed')
   const [assetsCache, setAssetsCache] = useState<EditVideoAssetsCache>({})
-
+  const [selectedVideoTabsCachedDirtyFormData, _setSelectedVideoTabsCachedDirtyFormData] = useState<
```

Comment:
This is `videoTabsCached...`, not `selectedVideoTabs` right? It's a record of all of them

Reply:
yup